apps/bajour/pages/search/index.tsx

```diff
@@ -0,0 +1,162 @@
+import {css, Pagination, styled, TextField} from '@mui/material'
+import {Button} from '@wepublish/ui'
+import {alignmentForTeaserBlock, ApiV1, Teaser} from '@wepublish/website'
+import {useRouter} from 'next/router'
+import {useEffect, useState} from 'react'
+import {MdSearch} from 'react-icons/md'
+
+export const SearchInput = styled(TextField)`
+  width: 100%;
+`
+
+export const SearchForm = styled('form')`
+  width: 90%;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: nowrap;
+`
+
+export const SearchButton = styled(Button)`
+  margin-left: ${({theme}) => theme.spacing(2)};
+`
+
+export const NavbarPhraseResultsWrapper = styled('div')`
+  padding: ${({theme}) => theme.spacing(2.5)};
+  margin-top: ${({theme}) => theme.spacing(6)};
+  background-color: ${({theme}) => theme.palette.common.white};
+  display: grid;
+  gap: ${({theme}) => theme.spacing(3)};
+  overflow-y: scroll;
+  padding-bottom: ${({theme}) => theme.spacing(10)};
+
+  ${({theme}) => css`
+    ${theme.breakpoints.up('md')} {
+      gap: ${theme.spacing(6)};
+      row-gap: ${theme.spacing(12)};
+      grid-template-columns: 1fr 1fr;
+      padding: ${theme.spacing(2.5)} calc(100% / 6) calc(100% / 12);
+    }
+  `}
+`
+
+export const SearchPageWrapper = styled('div')`
+  padding-top: ${({theme}) => theme.spacing(6)};
+`
+
+export const SearchStatus = styled('div')`
+  margin: 0 auto;
+  padding: ${({theme}) => theme.spacing(4)};
+  text-align: center;
+`
+
+export const SearchPagination = styled(Pagination)`
+  margin-top: 20px;
+`
+
+const ITEMS_PER_PAGE = 5
+
+const SearchPage: React.FC = () => {
+  const router = useRouter()
+  const {query} = router
+  const [rawQuery, setRawQuery] = useState((query.q as string) || '')
+  const [phraseQuery, setPhraseQuery] = useState((query.q as string) || '')
+  const [page, setPage] = useState(1)
+
+  const {
+    data: phraseData,
+    loading,
+    error
+  } = ApiV1.usePhraseQuery({
+    variables: {
+      query: phraseQuery,
+      take: ITEMS_PER_PAGE,
+      skip: (page - 1) * ITEMS_PER_PAGE
+    }
+  })
+
+  useEffect(() => {
+    if (query.q) {
+      setPhraseQuery(query.q as string)
+    }
+  }, [query.q])
+
+  const handleSearch = (e: React.FormEvent) => {
```

Comment:
Please import the types and not rely on a global hidden import (React) that comes from a babel plugin/compiler.